src/row_derive.md

```diff
@@ -0,0 +1,41 @@
+Derive macro for [`Row`][trait@crate::Row].
```

Comment:
nit: perhaps we should keep everything in the README, so it ends up on the "title" page on the crates.io too.